packages/rx_bloc_cli/lib/src/commands/create_command.dart

```diff
@@ -46,6 +46,7 @@ class CreateCommand extends Command<int> {
   final _projectNameString = 'project-name';
   final _organisationString = 'organisation';
   final _analyticsString = 'enable-analytics';
+  final _counterString = 'feature_counter';
```

Comment:
What do you think of the following param name: enable-feature-counter ?

Reply:
Sounds better 👍